README.md

```diff
@@ -15,7 +15,7 @@ CLI tool to generate github contributors hall of fame for github readme
 ## Install
 
 ```shell
-$ npm i -g rhof-cli
+$npm -g i @gh-conf/rhof
```

Comment:
it should be 
```shell
$ npm i -g rhof
```
Name of the package is `rhof`
`@gh-conf/rhof` is the base package used in this library

Reply:
It specifies on npm to write`@gh-conf/rhof`

Made the changes nevertheless

Reply:
This is the correct npm url
https://www.npmjs.com/package/rhof for this package

Reply:
Then it is showing the wrong install code on npm as well.

Reply:
Can you share what is the error you are getting on running this 
`npm i -g rhof`
?